app/models/case_log.rb

```diff
@@ -568,12 +567,12 @@ def age_under_16?(person_num)
   end
 
   def process_postcode_changes!
-    self.postcode_full = upcase_and_remove_whitespace(postcode_full)
+    self.postcode_full = format_postcode(postcode_full)
```

Comment:
I think this is where data is being processed before saved in DB, so no changes needed here.